app/backplane/power_module/src/main.cpp

```diff
@@ -8,9 +8,10 @@
 #include <f_core/os/c_task.h>
 #include <f_core/os/n_rtos.h>
 
+LOG_MODULE_REGISTER(main);
 
 int main() {
-    CPowerModule powerModule{};
+    static CPowerModule powerModule{};
```

Comment:
Threads are expected to last the entire lifetime and when main exits, that memory will get destroyed. When we run on actual hardware, we get stuck on an irq_lock after the RTOS is started because threads go out of memory. This wasn't caught in simulation since we sleep until the rtos stops